package.json

```diff
@@ -108,6 +108,8 @@
     "typescript": "^4.0.3"
   },
   "dependencies": {
-    "clsx": "^1.1.0"
+    "@types/tinycolor2": "^1.4.2",
+    "clsx": "^1.1.0",
```

Comment:
`clsx` is already in material - so don't need to add explicitly

Reply:
I haven't added it here but happy to remove.

Reply:
ok - leave it in we do  use it in one place, so better be explicit in case material remove it